lib/sidekiq/ultimate/expirable_set.rb

```diff
@@ -52,7 +52,7 @@ def initialize
       # @return [ExpirableSet] self
       def add(element, ttl:)
         @mon.synchronize do
-          expires_at = Concurrent.monotonic_time + ttl
+          expires_at = Process.clock_gettime(Process::CLOCK_MONOTONIC) + ttl
```

Comment:
Concurrent.monotonic_time does the same, yet looks cleaner. Sidekiq depends on concurrent gem. Why do we change this?

Reply:
Yes, `Concurrent.monotonic_time` is the same indeed. It was suggested by rubocop 😄

Reply:
LOL